okta provider crashes when trying to data "okta_auth_server"

Encountered this error which prevents me from working with okta provider

`Error: Plugin did not respond
│ 
│   with module.main-api-gateway-sandbox-cert.data.okta_auth_server.get_auth_server_id,
│   on ../main-api-gateway-rest-sandbox-cert/main.tf line 17, in data "okta_auth_server" "get_auth_server_id":
│   17: data "okta_auth_server" "get_auth_server_id" {
│ 
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ReadDataSource call. The plugin logs may contain more details.
╵
Releasing state lock. This may take a few moments...

Stack trace from the terraform-provider-okta_v3.13.13 plugin:

panic: runtime error: index out of range [0] with length 0

goroutine 60 [running]:
github.com/okta/terraform-provider-okta/okta.dataSourceAuthServerRead({0x1a64388, 0xc0004cc840}, 0x21a8a68, {0x1829280, 0xc000482270})
        github.com/okta/terraform-provider-okta/okta/data_source_okta_auth_server.go:69 +0x55f
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).read(0xc00048bdc0, {0x1a64318, 0xc00019e700}, 0xc00037a690, {0x1829280, 0xc000482270})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.1/helper/schema/resource.go:347 +0x12e
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).ReadDataApply(0xc00048bdc0, {0x1a64318, 0xc00019e700}, 0xc000309e00, {0x1829280, 0xc000482270})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.1/helper/schema/resource.go:558 +0xf7
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ReadDataSource(0xc0003e7788, {0x1a64318, 0xc00019e700}, 0xc000309c60)
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.1/helper/schema/grpc_provider.go:1105 +0x32c
github.com/hashicorp/terraform-plugin-go/tfprotov5/server.(*server).ReadDataSource(0xc000490e00, {0x1a643c0, 0xc000551d10}, 0xc000828870)
        github.com/hashicorp/terraform-plugin-go@v0.3.0/tfprotov5/server/server.go:247 +0x1db
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ReadDataSource_Handler({0x18f2100, 0xc000490e00}, {0x1a643c0, 0xc000551d10}, 0xc0004cc5a0, 0x0)
        github.com/hashicorp/terraform-plugin-go@v0.3.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:416 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0001d5500, {0x1a73250, 0xc000138780}, 0xc000284500, 0xc0004be360, 0x1fc2d90, 0x0)
        google.golang.org/grpc@v1.32.0/server.go:1194 +0xc8f
google.golang.org/grpc.(*Server).handleStream(0xc0001d5500, {0x1a73250, 0xc000138780}, 0xc000284500, 0x0)
        google.golang.org/grpc@v1.32.0/server.go:1517 +0xa2a
google.golang.org/grpc.(*Server).serveStreams.func1.2()
        google.golang.org/grpc@v1.32.0/server.go:859 +0x98
created by google.golang.org/grpc.(*Server).serveStreams.func1
        google.golang.org/grpc@v1.32.0/server.go:857 +0x294

Error: The terraform-provider-okta_v3.13.13 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.`

--- a/okta/data_source_okta_auth_server.go
+++ b/okta/data_source_okta_auth_server.go
@@ -64,7 +64,7 @@ func dataSourceAuthServerRead(ctx context.Context, d *schema.ResourceData, m int
 		return diag.Errorf("failed to find auth server '%s': %v", name, err)
 	}
 	if len(servers) < 1 || servers[0].Name != name {
-		diag.Errorf("authorization server with name '%s' does not exist", name)
+		return diag.Errorf("authorization server with name '%s' does not exist", name)
 	}
 	authServer := servers[0]
 	d.SetId(authServer.Id)

